Sources/xcproj/PBXProj+Helpers.swift

```diff
@@ -110,6 +110,17 @@ extension PBXProj {
             return type?.rawValue
         }
     }
+    
+    /// Returns the object with the given configuration list (project or target)
+    ///
+    /// - Parameter reference: configuration list reference.
+    /// - Returns: target or project with the given configuration list.
+    func objectWithConfigurationList(reference: String) -> PBXObject? {
```

Comment:
Should this be under objects, as #84 moves all these helpers

Reply:
Do you mean [this PR](https://github.com/xcodeswift/xcproj/pull/184/files)? Yeah it should be there. My idea was to merge that PR first and then update this one moving the helper under objects.

Reply:
Yeah, whoops, meant that one

Reply:
Nice, I'll merge that one first and move this function before merging 👍